solr/modules/opentelemetry/build.gradle

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
+description = 'Open Telemetry (OTEL) tracer'
+
+dependencies {
+  implementation project(':solr:core')
+
+  // TODO: Prune down dependencies to the absolute necessary
+
+  implementation platform('io.opentelemetry:opentelemetry-bom-alpha')
+
+  implementation 'io.opentelemetry:opentelemetry-sdk-trace'
+  implementation 'io.opentelemetry:opentelemetry-sdk-extension-autoconfigure'
+  implementation 'io.opentelemetry:opentelemetry-opentracing-shim'
+  implementation 'io.opentelemetry:opentelemetry-semconv'
+  implementation 'io.opentelemetry:opentelemetry-exporter-otlp'
+  implementation 'io.opentelemetry:opentelemetry-exporter-jaeger'
+  implementation 'io.opentelemetry:opentelemetry-exporter-zipkin'
+
+  // gRPC transport via netty - since we already ship netty this is more lightweight than netty-shaded
+  runtimeOnly 'io.grpc:grpc-netty'
+  implementation 'io.grpc:grpc-protobuf'
+  implementation 'io.grpc:grpc-stub'
+  implementation 'io.grpc:grpc-context'
+  // See https://issues.apache.org/jira/browse/LOG4J2-3609 due to needing these annotations
+  compileOnly 'org.apache.tomcat:annotations-api'
+
+  // TODO: Both this and jaegertracer-configurator pull in com.squareup.okhttp3:okhttp HTTP client and kotlin stdlib. They get duplicated in each module, so consider declaring these explicitly on solr-core
```

Comment:
Pointing out this is okay but I think adding to solr-core just because 2 modules use them is not the right direction.  I'd rather see another approach or just accept it.  The jaeger one is going away.